### PR TITLE
group transfer to x-k8s.io

### LINKED ISSUE #11: transfer group from `gke-internal.googlesource.com` to `x-k8s.io`
As the title
/kind bug

### PR DESCRIPTION
**Pull request by denkensk**
/kind bug
fix #11

### CODE REVIEW

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: kueue
-  newTag: latest
+  newName: registry.cn-hangzhou.aliyuncs.com/kubernetes-image-hub/kube-queue
```
Comment (ahg-g) [inline config/manager/kustomization.yaml:15]: is this intentional? lets keep it as is.

Comment (denkensk) [reply config/manager/kustomization.yaml]: Sorry. It's a mistake. Fixed.

Comment (denkensk) [reply config/manager/kustomization.yaml]: It's auto-edit ...... Sorry

**Pull request merged**